src/main/java/ddingdong/ddingdongBE/domain/form/repository/FormFieldRepository.java

```diff
@@ -2,12 +2,27 @@
 
 import ddingdong.ddingdongBE.domain.form.entity.Form;
 import ddingdong.ddingdongBE.domain.form.entity.FormField;
+import ddingdong.ddingdongBE.domain.form.repository.dto.FieldListInfo;
 import java.util.List;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
 
 public interface FormFieldRepository extends JpaRepository<FormField, Long> {
 
     List<FormField> findAllByForm(Form form);
 
-    List<FormField> findAllByFormAndSection(Form form, String section);
+    @Query(value = """
+            SELECT f.id AS id, f.question AS question, f.field_type AS type, f.section AS section, COUNT(fa.id) AS count
+            FROM (
+                    SELECT *
+                    FROM form_field field
+                    WHERE field.form_id = :formId
+            ) AS f
+            LEFT JOIN form_answer AS fa
+            ON fa.field_id = f.id
+            GROUP BY f.id
+            ORDER BY f.id
+            """, nativeQuery = true)
```

Comment:
p2)
해당 쿼리에서 불필요하게 서브쿼리를 사용하는 거 같은데 그냥 조인해도 괜찮을거 같습니다!
```suggestion
    @Query(value = """
            SELECT 
		    f.id,
		    f.question,
		    f.field_type AS type,
		    f.section,
		    COUNT(fa.id) AS count
	   FROM form_field f
	   LEFT JOIN form_answer fa ON fa.field_id = f.id
	   WHERE f.form_id = :formId
	   GROUP BY f.id  -- id만으로도 충분
	   ORDER BY f.id
            """, nativeQuery = true)
```